bin/iuf-helm-upload

```diff
@@ -174,4 +173,6 @@ do
         err=1
     fi
 done
+
+echo "INFO 'iuf-helm-upload' script completed with exit code: $err"
```

Comment:
i would probably check the error code and provide a contextual message.

if $err == 0 -> INFO iuf-helm-upload script completed successfully
else: ERROR iuf-helm-upload script failed

also, there's no context in this message, so it would be helpful to add the context provided by the parameter passing